main/main.cpp

```diff
@@ -389,7 +394,12 @@ void Main::print_help(const char *p_binary) {
 	OS::get_singleton()->print("  -e, --editor                      Start the editor instead of running the scene.\n");
 	OS::get_singleton()->print("  -p, --project-manager             Start the project manager, even if a project is auto-detected.\n");
 	OS::get_singleton()->print("  --debug-server <uri>              Start the editor debug server (<protocol>://<host/IP>[:<port>], e.g. tcp://127.0.0.1:6007)\n");
-#endif
+#ifdef MODULE_GDSCRIPT_ENABLED
+#ifndef GDSCRIPT_NO_LSP
+	OS::get_singleton()->print("  --lsp-port <port>                 Use the port specified for the language server protocol instead of the one in the editor settings. The port must be between 0 to 65535.\n");
```

Comment:
Matter of taste, so feel free to ignore if you disagree:
```suggestion
	OS::get_singleton()->print("  --lsp-port <port>                 Use the specified port for the language server protocol. The port must be between 0 to 65535.\n");
```

I think keeping the lines short is important for the `--help` output, or it might have lines longer than the terminal columns.

Reply:
I agree, I didn't know how to best shorten it. I think this captures the usage succinctly